src/bindings/Vscode.ml

```diff
@@ -407,3 +415,83 @@ module LanguageClient = struct
     let open Promise.O in
     onReady t >>| fun () -> t.initializeResult
 end
+
+module ShellExecution = struct
+  type shellQuotingOptions
```

Comment:
How do we create values of this type?

Reply:
for now we don't, I didn't bind this part